### PR TITLE
added tutorial (init-tutorial.md) and its relevant image files

### PR DESCRIPTION
**Pull request by syuii**
Not sure yet if the images work in the md file and may still need changes for more detailed parameter descriptions.

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -8,5 +8,5 @@ chapters:
 - file: quantum-networks
 - file: what-is-quisp
 - file: getting-started
-- file: vscode
```
Comment (Tatzhiro) [inline _toc.yml:11]: You shouldn't be deleting other files

**Review by Tatzhiro — CHANGES_REQUESTED** [review]

**Review by Tatzhiro — CHANGES_REQUESTED** [review]
You should create an image folder or something and store all the png files there to keep the repository organized.
Change the path to the image file in your init-tutorial.md afterwards.

**Comment by sierpinskiii** [comment]
Restored the deleted file. Everything else is fine as I think of. Nice work!

**Comment by sierpinskiii** [comment]
Gotta restructure our directories including the image dir...

**Pull request merged**